src/WinGet.RestSource/Helpers/AppConfig/WinGetAppConfig.cs

```diff
@@ -0,0 +1,243 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="WinGetAppConfig.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.WinGet.RestSource.Helpers.AppConfig
```

Comment:
I would prefer to have this in its own separate project. We might have scenarios where we just want to add a dependency to use feature flags, but not the core project.